ast-visual-studio-extension/CxExtension/Panels/ResultsTreePanel.cs

```diff
@@ -122,6 +122,12 @@ private TreeViewItem BuildTree()
         // Convert AST results to tree view item
         private List<TreeViewItem> ConvertResultsToTreeViewItem(Results results)
         {
+
+            // Check if results or results.results is null and return an empty list if so
+            if (results?.results == null)
+            {
+                return new List<TreeViewItem>();
```

Comment:
Should we not display a msg: "Checkmarx found no vulnerabilities" as we have in VSCode?

Reply:
now the msg is:
e.g:

"No results for scan e9e9a8d1-b18b-4864-9292-0d98c327f2c3"